src/test/resources/org/jetbrains/research/ml/coding/assistant/tree/data/loop/out_0.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
```

Comment:
shouldn't this file be in a 'for' subdirectory?

Reply:
The same answer as the previous comment